client/clientservice/src/main.cpp

```diff
@@ -57,6 +57,7 @@ po::variables_map parseCmdLine(int argc, char** argv) {
     ("metrics-port", po::value<int>()->default_value(9891), "Prometheus port to query clientservice metrics")
     ("secrets-url", po::value<std::string>(), "URL to decrypt private keys")
     ("jaeger", po::value<std::string>(), "Push trace data to this Jaeger Agent")
+    ("max-receive-msg-size", po::value<int>()->default_value(52428800), "Clientservice max receive message size in bytes")
```

Comment:
Ideally, the default would be a much smaller value (such as 4MB or even lower) and during deployment, we decide how big the value should be. I don't know about the gRPC internals so this may or may not lead to unnecessary memory allocations.

Reply:
Updated.